base/uk.ac.stfc.isis.ibex.opis/resources/stage/motion_setpoint.opi

```diff
@@ -0,0 +1,1941 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Target, Current and Moving are column headers so are correct. Corrected others and now have 'offset:' is not sentence case this is ok because it is preceeded by pv so should be lower case.